### PR TITLE
Complete Android Release Script

### PR DESCRIPTION
**Pull request by jcnoriega**
jira ticket: https://engineering.paypalcorp.com/jira/browse/DTNOR-330
### Summary of changes

 - Added Core and Card publication on the GA script.
 - Version name in Core and Card where added
 - Tested releasing and it works!

 ### Checklist

 - [ ] Added a changelog entry

-

### CODE REVIEW

**Review by andrespelaezp — APPROVED** [review]
LGTM

**Review by sshropshire — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -122,4 +138,30 @@ jobs:
           sonatype_pwd: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
           signing_key_id: ${{ secrets.SIGNING_KEY_ID }}
           signing_key_pwd: ${{ secrets.SIGNING_KEY_PASSWORD }}
+          signing_key_file: ${{ env.SIGNING_KEY_FILE_PATH }}
```
Comment (sarahkoop) [inline .github/workflows/release.yml:141]: Have these all been updated to a generic PayPal user (not anyone's individual username and password?)

Comment (jcnoriega) [reply .github/workflows/release.yml:141]: no, not yet, maybe we can put a ticket for that. Once we get a user, we just update the secrets

Comment (sarahkoop) [reply .github/workflows/release.yml:141]: I personally think that should be a part of this ticket. We don't want to be releasing under individual sonatype accounts.

**Review by sarahkoop — APPROVED** [review]

**Pull request merged**